src/App.js

```diff
@@ -8,59 +8,67 @@ function App() {
 	const [city, setCity] = useState('New York City');
 	const [results, setResults] = useState(null);
 
-	useEffect(() => {
-		fetch(
-			'https://api.openweathermap.org/data/2.5/weather?q=' +
-				city +
-				'&units=metric' +
-				'&appid=' +
-				process.env.REACT_APP_APIKEY,
-		)
-			.then((res) => res.json())
-			.then(
-				(result) => {
-					if (result['cod'] !== 200) {
-						setIsLoaded(false);
-					} else {
-						setIsLoaded(true);
-						setResults(result);
-					}
-				},
-				(error) => {
-					setIsLoaded(true);
-					setError(error);
-				},
-			);
-	}, [city]);
+  let weatherConditions = new Map([
```

Comment:
I think we should add sunny weather too. I see you already added songs for common weather conditions. I am not sure what the spotify API offers but can we explore more options as we currently have with the items to bring functionality.?
Also, as mentioned by others, does the API allow us to pick from a list of playlist? However, we will work based on the functionalities of the API

Reply:
In weather API, there's no sunny condition as far as i saw. They included it as "Clear".